sqlmesh/core/snapshot/categorizer.py

```diff
@@ -35,9 +36,16 @@ def categorize_change(
         SnapshotChangeCategory.BREAKING if mode == AutoCategorizationMode.FULL else None
     )
 
-    if type(new_model) != type(old_model) or new.fingerprint.data_hash == old.fingerprint.data_hash:
+    if type(new_model) != type(old_model):
```

Comment:
Should this actually be explicitly returning breaking? I see None is the other option but if the types changed ins't that definitely breaking?

Reply:
if the auto categorizer is off, None is used to let the user choose, we shouldn't do any categorization is if the categorize is off is how i interpret it

Reply:
Ah thanks I got it now